crates/red_knot_python_semantic/resources/mdtest/type_properties/is_equivalent_to.md

```diff
@@ -32,4 +32,22 @@ static_assert(not is_equivalent_to(Literal[1, 0], Literal[1, 2]))
 static_assert(not is_equivalent_to(Literal[1, 2, 3], Literal[1, 2]))
 ```
 
+## Differently ordered intersections and unions are equivalent
+
+```py
+from knot_extensions import is_equivalent_to, static_assert, Intersection, Not
+
+class P: ...
+class Q: ...
+class R: ...
+class S: ...
+
+static_assert(is_equivalent_to(int | str, str | int))
```

Comment:
Should we maybe write this using `P` and `Q` as well`? And potentially do it for unions with three elements as well?
```suggestion
static_assert(is_equivalent_to(P | Q, Q | P))

static_assert(is_equivalent_to(P | Q | R, P | R | Q))
static_assert(is_equivalent_to(P | Q | R, R | P | Q))
static_assert(is_equivalent_to(P | Q | R, R | Q | P))
static_assert(is_equivalent_to(P | Q | R, Q | P | R))
static_assert(is_equivalent_to(P | Q | R, Q | R | P))
```